src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -121,6 +121,12 @@ public Response enrollSDKinRelease(@PathParam("releaseName") String releaseName,
     }
 
     Release release = RELEASEDAO.getRelease(platform, releaseName);
+    long currentTime = System.currentTimeMillis() / 1000;
+    // Check if the release is old and if so then dont allow enrollment.
+    if (currentTime > release.codeFreezeTime()) {
```

Comment:
nit: Adding a suffix of Millis or Ms on both these Time variables would make it easier to see that they are both in the same units. But this would need changes in all the places where the codeFreezeTime and other Time variables are used. Feel free to add a TODO for this too.